### ISSUE TITLE
FFT tests causing memory corruption

### ISSUE THREAD

**Issue by leto** [leto/math--gsl#131]
Seems to happen rarely on OS X but every time on Linux:

```
# Math::GSL::FFT::Test->FFT_COMPLEX_RADIX2_DIF_FORWARD
*** Error in `/home/travis/perl5/perlbrew/perls/5.24/bin/perl': free(): invalid next size (fast): 0x0000000002a567b0 ***
```

from https://s3.amazonaws.com/archive.travis-ci.org/jobs/165875627/log.txt

**Comment by polettix** [comment]
The linked log is inaccessible (maybe because it's from three years ago?)

**Comment by leto** [comment]
@polettix yeah, it was probably deleted. If you can submit a documentation or test patch that improves FFT in any way, that would be good appreciated. Triggering this bug might be hard and/or it might not exist anymore
